package.json

```diff
@@ -58,7 +58,7 @@
     "mute-stream": "0.0.5",
     "open": "0.0.5",
     "osenv": "0.1.3",
-    "pbxproj-dom": "1.0.3",
+    "pbxproj-dom": "^1.0.7",
```

Comment:
This is not very safe - in case `pbxproj-dom` is released with breaking change or some unexpected behavior for us in a minor version, we'll be broken on live. That's why we prefer strict versions in package.json